### PR TITLE
Increase batch size in background thread transport

### LINKED ISSUE #85: Increase batch size in background thread transport
Increase batch size and do not wait between each export to ensure all spans are sent within the grace period when terminating the thread.

### PR DESCRIPTION
**Pull request by liyanhui1228**
Closes #85

### CODE REVIEW

File: trace/opencensus/trace/exporters/transports/background_thread.py [hunk 1]
```diff
@@ -107,14 +106,12 @@ def _thread_main(self):
...
                     'spans': spans,
                 }
+
                 self.exporter.emit(spans_json)
 
             for _ in range(len(items)):
                 self._queue.task_done()
 
-            # Wait for a while before next export
-            time.sleep(_WAIT_PERIOD)
```
Comment (chingor13) [inline trace/opencensus/trace/exporters/transports/background_thread.py]: Will removing the sleep make this into a busy loop and eat CPU?

File: trace/opencensus/trace/exporters/transports/background_thread.py [hunk 2]
```diff
@@ -158,11 +155,14 @@ def stop(self):
             return True
 
         with self._lock:
+            print('Number of spans pending sent: {}'.format(
```
Comment (chingor13) [inline trace/opencensus/trace/exporters/transports/background_thread.py]: Is the printing meant to stay or is it just for debugging?

If intentional, should it be some kind of optional, configurable logger instead?

File: trace/opencensus/trace/exporters/transports/background_thread.py (hunk 1 above)
Comment (liyanhui1228) [reply trace/opencensus/trace/exporters/transports/background_thread.py]: Yeah agree, changed to reduce the sleep time to 1 second. Just want to make sure the spans are sent within the grace period.

File: trace/opencensus/trace/exporters/transports/background_thread.py (hunk 2 above)
Comment (liyanhui1228) [reply trace/opencensus/trace/exporters/transports/background_thread.py]: Removed.

**Pull request merged**